packages/buidler-core/sample-project/scripts/sample-script.js

```diff
@@ -1,22 +1,13 @@
-// We require the Buidler Runtime Environment explicitly here. This is optional
```

Comment:
I guess this is the bit you were referring to with your comment about extending the BRE, but I don't get it. This isn't an extension, it's just plain usage and this worked as an example of how to use it directly, which is basic Buidler usage. Why remove it?

Reply:
As I mentioned before, I asked @alcuadrado if I could remove this in order to simplify the script. There are two main reasons:

- Extending the BRE is an advanced use and is explained in /advanced/buidler-runtime-environment.html
- When running `npx buidler run scripts/sample-script.js` the contracts were compiling twice.